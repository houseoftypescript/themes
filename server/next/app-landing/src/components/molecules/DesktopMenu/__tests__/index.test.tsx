import { render } from '@testing-library/react';
import { DesktopMenu } from '../index';

describe('DesktopMenu', () => {
  it('to match snapshot', () => {
    const { container } = render(<DesktopMenu scrolled />);
    expect(container).toMatchSnapshot();
  });
});
