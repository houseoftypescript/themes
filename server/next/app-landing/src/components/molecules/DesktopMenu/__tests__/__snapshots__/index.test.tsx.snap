// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DesktopMenu to match snapshot 1`] = `
<div>
  <div
    class="flex items-center gap-8"
  >
    <a
      class="cursor-pointer font-medium capitalize tracking-wider"
    >
      Services
    </a>
    <a
      class="cursor-pointer font-medium capitalize tracking-wider"
    >
      projects
    </a>
    <a
      class="cursor-pointer font-medium capitalize tracking-wider"
    >
      pricing
    </a>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-11ogog7-MuiButtonBase-root-MuiButton-root"
      tabindex="0"
      type="button"
    >
      Download
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </button>
  </div>
</div>
`;
